packages/core/data-transfer/lib/engine/index.ts

```diff
@@ -351,6 +350,8 @@ class TransferEngine<
         );
       }
 
+      this.#emitTransferUpdate('start');
```

Comment:
I also question if we should rename these events to match the actual stages. So we could have bootstrap, init, beforeTransfer, and maybe even the transfer stages as transferUpdate events in addition to stageUpdate events. Or merge them into one event type. Thoughts?

Reply:
Even if I do agree it could be nice to have more detailed events, do you see a real use case for them at the moment?
If not, we better wait for a real use case and then implement them rather than trying to anticipate & create an API we'll have to break.
What do you think?

Reply:
That's the thought actually, this is our chance to break them without causing any problems. Do we want to stick with these names, or do we want them to match what we're doing? I guess the question is if they should be somewhat synced with the transfer stages, or if they're independent (init / start / finish / error) regardless of what stages we add